Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -681,9 +681,9 @@ function_exists('posix_isatty'),
 
             if (class_exists('Symfony\Component\Intl\Intl')) {
                 $this->addRecommendation(
-                    \Symfony\Component\Intl\Intl::getIcuDataVersion() === \Symfony\Component\Intl\Intl::getIcuVersion(),
-                    sprintf('intl ICU version installed on your system (%s) should match the ICU data bundled with Symfony (%s)', \Symfony\Component\Intl\Intl::getIcuVersion(), \Symfony\Component\Intl\Intl::getIcuDataVersion()),
-                    'In most cases you should be fine, but please verify there is no inconsistencies between data provided by Symfony and the intl extension. See https://github.com/symfony/symfony/issues/15007 for an example of inconsistencies you might run into.'
+                    \Symfony\Component\Intl\Intl::getIcuDataVersion() <= \Symfony\Component\Intl\Intl::getIcuVersion(),
```

Comment:
Ok, what we're dealing with here is actually two problems:
* the system version of the intl extension is outdated
* the system version of the intl extension doesn't match the one bundled with Symfony

I've added a second check to suggest Symfony update if the version installed on the system is newer.